ext/ffi/ffi.c

```diff
@@ -5301,6 +5302,19 @@ static zend_result zend_ffi_preload(char *preload) /* {{{ */
 }
 /* }}} */
 
+static zend_result (*prev_zend_post_startup_cb)(void);
+static zend_result ffi_fixup_temporaries(void) {
+	if (ZEND_OBSERVER_ENABLED) {
```

Comment:
I think this could really benefit from a code comment, as to *why* this is needed, and what it does?